src/rust/engine/fs/src/lib.rs

```diff
@@ -207,7 +207,7 @@ pub enum PathGlob {
 #[derive(Clone, Debug, Eq, Hash, PartialEq)]
 pub struct GlobParsedSource(String);
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Necessary to get this working in the test:

```rust
assert_eq!(
      pg.include,
      PathGlob::spread_filespecs(&include_globs).expect("Include globs failed to expand")
    );
```

`GlobParsedSource` implements Eq, `PathGlob` implements Eq, and Vec<T> implements PartialEq when T does as well. So, I think this is safe to change.

Reply:
I feel like I would generally expect `Eq` to also be derived in cases where it can be auto-derived. cc @illicitonion @pierrechevalier83 @stuhood thoughts?